src/CSModule.sol

```diff
@@ -1668,7 +1668,6 @@ contract CSModule is
         no.stuckValidatorsCount = uint32(stuckValidatorsCount);
         emit StuckSigningKeysCountChanged(nodeOperatorId, stuckValidatorsCount);
 
-        // TODO: think about reforge the condition: is `no.depositableValidatorsCount > 0` required?
```

Comment:
why?

Reply:
This condition is needed to avoid unnecessary calls inside the block